pkg/types/types.go

```diff
@@ -270,14 +275,32 @@ func (s byStepTime) Less(i, j int) bool {
 	return s[i].StepTime < s[j].StepTime
 }
 
-func mergeMetrics(metrics []Metric, mismatchCheck bool) (metric Metric, mismatches int) {
+func getPointMajorityValue(values []float64) (majorityValue float64, majorityCount int) {
+	valuesCount := len(values)
+	if valuesCount == 0 {
+		return 0, 0
```

Comment:
This does not seem like a correct behaviour. No values does not give a zero value.

Reply:
It is never called while a mismatch isn't seen. Anyways, as this is also returning `majorityCount`, It should be obvious for the caller not to use the majorityValue while no supporting value is there (zero count). WDYT?

Reply:
Made the change.